lib/routes/password.js

```diff
@@ -149,7 +149,7 @@ module.exports = function (
         var wantsKeys = requestHelper.wantsKeys(request)
         var account, verifyHash, sessionToken, keyFetchToken
 
-        return changePassword()
+        changePassword()
```

Comment:
No worries - my only concerns was if we needed it for e.g. tests, but if it's not breaking anything, I don't really mind :-)